Assignment-10-August-22/userprofile/src/containers/Pages/Login.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useNavigate, Link } from "react-router-dom";
+import { FormGroup, FormControl } from "@mui/material";
+import { useForm, Controller } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as Yup from "yup";
+
+import { login } from "../../Store/authSlice";
+import ButtonComponent from "../../Components/ButtonComponent";
+import InputField from "../../Components/InputField";
+
+const validationSchema = Yup.object({
+  email: Yup.string().email("Invalid email format").required("Email is required"),
+  password: Yup.string().min(6, "Password must be at least 6 characters").required("Password is required"),
+});
+
+export default function Login() {
+  const userArray = useSelector((state) => state.userArray.value);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
```

Comment:
I don't think you are setting document title anywhere. Make a custom function to handle that.